src/torchmetrics/functional/text/bert.py

```diff
@@ -257,9 +257,101 @@ def _rescale_metrics_with_baseline(
     return all_metrics[..., 0], all_metrics[..., 1], all_metrics[..., 2]
 
 
+def _preprocess_multiple_references(
+    preds: List[str],
+    target: List[Union[str, Sequence[str]]],
+    ref_group_boundaries: Optional[list[tuple[int, int]]],
+) -> tuple[List[str], List[str], Optional[List[tuple[int, int]]]]:
+    """Preprocesses predictions and targets when dealing with multiple references.
+
+    This function handles the case where a single prediction might have multiple
+    reference targets (represented as a list/tuple of strings).
+
+    Args:
+        preds: A list of predictions
+        target: A list of targets, where each item could be a string or a list/tuple of strings
+        ref_group_boundaries: `None`
```

Comment:
please add description of `ref_group_boundaries`, or is it always none? because then lets have that as default value